client/app/telecom/pack/xdsl/modem/connectedDevices/pack-xdsl-modem-connectedDevices.html

```diff
@@ -36,34 +36,23 @@
             </button>
         </div>
 
-
-        <div class="description table-responsive mt-2" data-ng-show="DeviceCtrl.showDevices">
-            <table class="table table-hover table-condensed">
-                <thead>
-                <tr>
-                    <th scope="col"></th>
-                    <th scope="col" data-translate="xdsl_modem_device_mac"></th>
-                    <th scope="col" data-translate="xdsl_modem_device_ip"></th>
-                    <th scope="col" data-translate="xdsl_modem_device_host"></th>
-                </tr>
-                </thead>
-                <tbody>
-                <tr data-ng-repeat="device in DeviceCtrl.devices track by device.macAddress">
-                    <td>
-                        <span class="ovh-font h5"
-                           data-ng-class="DeviceCtrl.getConnectionIcon(device)"
-                           title="{{device.interfaceType}}"></span>
-                        <span class="ovh-font h5"
-                           data-ng-class="DeviceCtrl.getDeviceInfo(device).icon"
-                           title="{{DeviceCtrl.getDeviceInfo(device).tooltip | translate}}"></span>
-                    </td>
-                    <th scope="row" data-translate-attr="{ title : 'xdsl_modem_device_mac' }" data-ng-bind="device.macAddress"></th>
-                    <td data-translate-attr="{ title : 'xdsl_modem_device_ip' }" data-ng-bind="device.ipAddress"></td>
-                    <td data-translate-attr="{ title : 'xdsl_modem_device_host' }" data-ng-bind="device.hostName"></td>
-                </tr>
-                </tbody>
-            </table>
+        <div class="description mt-2" data-ng-show="DeviceCtrl.showDevices">
+            <oui-datagrid rows="DeviceCtrl.devices">
+                <oui-column property="interfaceType" sortable> 
+                    <span class="ovh-font h5"
+                        data-ng-class="DeviceCtrl.getConnectionIcon($row)"
+                        title="{{$row.interfaceType}}"></span>
```

Comment:
```ng-bind``` would be better than title

Reply:
or use of `data-ng-attr-title` ?

Reply:
Why a ```span``` would have a title and no text?
https://jsfiddle.net/vg75jehq/